ui/app/adapters/kv/metadata.js

```diff
@@ -27,11 +32,14 @@ export default class KvMetadataAdapter extends ApplicationAdapter {
     });
   }
 
-  // TODO: replace this with raw request for metadata request?
   query(store, type, query) {
-    const { backend } = query;
-    return this.ajax(this._url(`${encodePath(backend)}/metadata?list=1`), 'GET').then((resp) => {
-      resp.data.backend = backend;
+    const { backend, secretPrefix } = query;
```

Comment:
Good thought. But yeah, secretPrefix is supposed to clarify that it is pre-secret. Meaning that it will never be the full secret path, but the directory structure. Eg. beep/boop/ but not beep/boop/bop.

Happy to hear other's thoughts as well.

Reply:
Maybe `pathToSecret`? The original `secretPath` made me think it was the full thing

Reply:
Oh, I like that. @hellobontempo what do you think about `pathToSecret`? If I get two votes, I'll go ahead and modify.

Reply:
Yeah, I like it! Good suggestion!

Reply:
Amended! Thanks for all the brainstorming!